README.md

```diff
@@ -24,7 +24,7 @@ Usage
 -----
 ### Initialize
 ```cs
-RazorpayClient client = new RazorpayClient(key, secret);
+RazorpayClient client = new RazorpayClient(key, secret, baseUrl);
```

Comment:
Should specify that here baseUrl is an optional parameter

Reply:
Ok, I will update.